tsconfig.base.json

```diff
@@ -22,13 +22,14 @@
 			"@ngx/layout": ["libs/layout/src/public-api.ts"],
 			"@ngx/store": ["libs/store/src/public-api.ts"],
 			"@ngx/table": ["libs/table/src/public-api.ts"],
+			"@ngx/tour": ["libs/tour/src/index.ts"],
 			"@ngx/utils": ["libs/utils/src/public-api.ts"],
-			"@ngx/tour": ["libs/tour/src/index.ts"]
```

Comment:
Why does this one not match the others?

Reply:
That's the new approach of Nx, it has been like that for quite a while now (was already the case with the ngx-tour package). I don't know if there's a migration script for it, but I'm hesitant to edit this if this is their new approach 😅

Reply:
Fair enough, maybe we should create a task on the board to do the same for the existing packages.